flytekit/bin/entrypoint.py

```diff
@@ -185,6 +180,18 @@ def _dispatch_execute(
         exit(1)
 
 
+def get_traceback_str(e: Exception) -> str:
+    tb = e.__cause__.__traceback__ if e.__cause__ else e.__traceback__
+    lines = traceback.format_tb(tb)
+    lines = [line.rstrip() for line in lines]
+    lines = "\n".join(lines).split("\n")
```

Comment:
Is this line required? It's joining on `\n` and then splitting it again.

Reply:
lol. It's copied from [here](https://github.com/flyteorg/flytekit/blob/4a97da54dde5b96b348dd38cf6bfd0af0549594b/flytekit/exceptions/scopes.py#L43-L46)

Reply:
I just removed that line